tests/store/tracking/test_file_store.py

```diff
@@ -885,6 +885,17 @@ def test_log_param_enforces_value_immutability(self):
         run = fs.get_run(run_id)
         assert run.data.params[param_name] == "value1"
 
+    def test_log_param_max_length_value(self):
+        param_name = "new param"
+        param_value = list(range(500))
+        fs = FileStore(self.test_root)
+        run_id = self.exp_data[FileStore.DEFAULT_EXPERIMENT_ID]["runs"][0]
+        fs.log_param(run_id, Param(param_name, param_value))
+        run = fs.get_run(run_id)
+        assert run.data.params[param_name] == str(param_value)
+        with pytest.raises(MlflowException, match="exceeded length"):
+            fs.log_param(run_id, Param(param_name, list(range(1000))))
```

Comment:
```suggestion
            fs.log_param(run_id, Param(param_name, "x" * 1000))
```

Reply:
yes, at this point, everything should have been stringified. Will change